profile.d/security

```diff
@@ -17,12 +17,12 @@ SIGN_ALIAS=jenkins
 
 ### Git Staging Promotion Settings
 #
-RELEASE_GIT_PRODUCTION_REPOSITORY=git@github.com:jenkinsci/jenkins.git
-RELEASE_GIT_PRODUCTION_BRANCH=master
+RELEASE_GIT_PRODUCTION_REPOSITORY=git@github.com:jenkinsci-cert/jenkins.git
+RELEASE_GIT_PRODUCTION_BRANCH=security-master
```

Comment:
This variable is used to promote git content from a source to destination, you never used that as you preferred to do it manually